.github/workflows/build.yml

```diff
@@ -12,6 +12,9 @@ env:
   DOMAIN:                london.cloudapps.digital
   APPLICATION:           Get Into Teaching API Service
   PAAS_APPLICATION_NAME: get-into-teaching-api
+  SLACK_FAILURE:         '#ff0000'
```

Comment:
I think we need an 'alert' colour here

eg 

Green = success
Red = Failed deploy
Amber = Testsuite failure on branch